### ISSUE TITLE
installation fails: No such file or directory: 'test-requirements.txt'

### ISSUE THREAD

**Issue by nskalis** [Juniper/py-junos-eznc#941]
Hi,

Install latest PyEz from PyPI fails.
Please let me know if you need additional information.

```
# pip3.6 install junos-eznc --upgrade
Collecting junos-eznc
  Using cached https://files.pythonhosted.org/packages/da/6c/6a9c839f2d7bef53381c84a7e49620b44a87633306832bc5e3bf7b68fe00/junos_eznc-2.2.1-py2.py3-none-any.whl
Collecting scp>=0.7.0 (from junos-eznc)
  Downloading https://files.pythonhosted.org/packages/4d/7a/3d76dc5ad8deea79642f50a572e1c057cb27e8b427f83781a2c05ce4e5b6/scp-0.13.2-py2.py3-none-any.whl
Requirement already up-to-date: six in /usr/local/lib/python3.6/site-packages (from junos-eznc)
Collecting lxml>=3.2.4 (from junos-eznc)
  Downloading https://files.pythonhosted.org/packages/35/8a/5e066949f2b40caac32c7b2a77da63ad304b5fbe869036cc3fe4a198f724/lxml-4.3.3-cp36-cp36m-manylinux1_x86_64.whl (5.7MB)
    100% |################################| 5.7MB 145kB/s 
Requirement already up-to-date: jinja2>=2.7.1 in /usr/local/lib64/python3.6/site-packages (from junos-eznc)
Requirement already up-to-date: pyserial in /usr/lib/python3.6/site-packages (from junos-eznc)
Collecting ncclient>=0.5.4 (from junos-eznc)
  Using cached https://files.pythonhosted.org/packages/d2/1f/4333dba7d63bebb27232d0cd5d2173ed14615f50c92a41479b86aab3217d/ncclient-0.6.5.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-27wxtdvc/ncclient/setup.py", line 37, in <module>
        test_req_lines = [line.strip() for line in open("test-requirements.txt").readlines()]
    FileNotFoundError: [Errno 2] No such file or directory: 'test-requirements.txt'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-27wxtdvc/ncclient/
```

**Comment by rsmekala** [comment]
@nskalis `ncclient` install is broken in the latest version:0.6.5. Please refer https://github.com/ncclient/ncclient/issues/321. 

Please use `pip3.7 install ncclient==0.6.4` and then install PyEZ as a workaround for the issue. I have verified it and it is working with `ncclient-0.6.4`.

**Comment by nskalis** [comment]
thanks!

**Comment by vnitinv** [comment]
This was fixed with the latest ncclient release of 0.6.6
